cdist/conf/type/__link/explorer/state

```diff
@@ -33,6 +33,7 @@ destination_dir="${destination%/*}"
 case "$type" in
    symbolic)
       cd "$destination_dir" || exit 1
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2012

Should we use something different here instead of ls?

The same for below TODOs.